docs/production-deployment/cloud/operation-api.mdx

```diff
@@ -14,20 +14,13 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- Temporal Cloud Operations API is in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for Temporal Cloud.
+Temporal Cloud Operations API is in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for Temporal Cloud.
 
 :::
 
 The [Temporal Cloud Operations API](https://github.com/temporalio/api-cloud/tree/main) (Cloud Ops API) is a library for managing the automation of Users, Namespaces, and Temporal Cloud Accounts.
 
-Cloud Ops API is an open-source, public gRPC API, library to compile and generate code in your desired programming language.
-
-## Overview
```

Comment:
I deleted this section because it reads like a duplication of the previous two paragraphs. I can put it put if we feel like it still serves some purpose, but I think we should at least delete the heading because the first paragraphs are already an overview.